### PR TITLE
adding script to performance test evaluate

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: tests/test_evalaute_api_performance.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                self.assertEqual(201, resp.status_code)
+                role_ids.append(role_id)
+            add_roles_url=furl(f'/v1/user/{user}/roles', query_params={'action': 'add'})
+            resp = self.app.put(add_roles_url.url,headers=headers, data=json.dumps({'roles':role_ids}))
+            resp.raise_for_status()
+            profile(user, repeat)(self._run_test)(user, headers, repeat)
+            # profileit(user)(self._run_test)(user, headers, repeat)
+            start = role
+
+    def test_lookup_roles(self):
```
Comment (amarjandu) [inline tests/test_evalaute_api_performance.py]: function is also defined above.

File: tests/test_evalaute_api_performance.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    "action": ["fus:GetUser"],
+                    "resource": [f"arn:hca:fus:*:*:user/{user}/policy"],
+                    "principal": user
+                })
+        for i in range(repeat):
+            resp = self.app.post('/v1/policies/evaluate', headers=headers, data=json_body)
+            resp.raise_for_status()
+
+    def test_lookup_roles(self):
+        # this looks at how lookup policy scales
```
Comment (amarjandu) [inline tests/test_evalaute_api_performance.py]: descriptive DocString

File: tests/test_evalaute_api_performance.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+from furl import furl
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from tests.base_api_test import BaseAPITest
+from tests.common import get_auth_header, service_accounts, create_test_statement
+
+import cProfile
+def profileit(name):
```
Comment (amarjandu) [inline tests/test_evalaute_api_performance.py]: DocString

File: tests/test_evalaute_api_performance.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            return retval
+        return wrapper
+    return inner
+
+
+import time
+import logging
+
+def profile(name, count):
+    def inner(func):
```
Comment (amarjandu) [inline tests/test_evalaute_api_performance.py]: DocString

File: tests/test_evalaute_api_performance.py [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+            add_groups_url=furl(f'/v1/user/{user}/groups', query_params={'action': 'add'})
+            resp = self.app.put(add_groups_url.url,headers=headers, data=json.dumps({'groups':group_ids}))
+            resp.raise_for_status()
+            profile(user, repeat)(self._run_test)(user, headers, repeat)
+            # profileit(user)(self._run_test)(user, headers, repeat)
+            start = group
+
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (amarjandu) [inline tests/test_evalaute_api_performance.py]: EOF newline

File: tests/test_evalaute_api_performance.py (hunk 1 above)
Comment (Bento007) [reply tests/test_evalaute_api_performance.py]: fixed

File: tests/test_evalaute_api_performance.py (hunk 2 above)
Comment (Bento007) [reply tests/test_evalaute_api_performance.py]: improved

File: tests/test_evalaute_api_performance.py (hunk 3 above)
Comment (Bento007) [reply tests/test_evalaute_api_performance.py]: removed

File: tests/test_evalaute_api_performance.py (hunk 4 above)
Comment (Bento007) [reply tests/test_evalaute_api_performance.py]: removed

File: tests/test_evalaute_api_performance.py (hunk 5 above)
Comment (Bento007) [reply tests/test_evalaute_api_performance.py]: fixed

**Comment by Bento007** [comment]
see #182

**Pull request closed without merging**